lua/types/init.lua

```diff
@@ -0,0 +1,5 @@
+local types = {}
+
+types.cn = require("types.cn")
```

Comment:
is `types.cn` module really necessary or can you just drop the code here?

Reply:
It is not strictly necessary in a manner of speaking...  Before i made this Pull Request, i checked for similar requests, and i found this along the way #1 . Which suggested other sources/modules. 

Therefore, i considered that there's a possibility in the future that the codebase could grow (expand modules or be reused separately), and it's better to keep them separate for better organization and maintainability with the different modules types like: `carbon-now`, `ray.so` or `bin-share`.

However, it makes more sense to keep it simple in one file for the time being... i could revert that if you want it, so LMK (let me know).

Reply:
> This one I do admit is my fault 😅 a little. I was too excited while doing it.

Reply:
> However, it makes more sense to keep it simple in one file for the time being... i could revert that if you want it, so LMK (let me know).

Yeah, just drop that module code over there and it should be fine. No need for that extra module

Reply:
Fixed on [8aa5de1](https://github.com/ellisonleao/carbon-now.nvim/pull/4/commits/8aa5de1d050c41f69bdedd20aebaff80a4f43313)